src/app/pages/rpc-stx-transfer-sip9-nft/rpc-stx-transfer-sip9-nft.routes.tsx

```diff
@@ -0,0 +1,35 @@
+import { Route } from 'react-router-dom';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { BroadcastErrorSheet } from '@app/components/broadcast-error-dialog/broadcast-error-dialog';
+import { CurrentStacksAccountLoader } from '@app/components/loaders/stacks-account-loader';
+import { FeeEditor } from '@app/features/fee-editor/fee-editor';
+import { ledgerStacksTxSigningRoutes } from '@app/features/ledger/flows/stacks-tx-signing/ledger-sign-stacks-tx-container';
+import { NonceEditor } from '@app/features/nonce-editor/nonce-editor';
+import { AccountGate } from '@app/routes/account-gate';
+import { LedgerStacksGate } from '@app/routes/ledger-stacks-gate';
+
+import { RpcStxTransferSip9Nft } from './rpc-stx-transfer-sip9-nft';
+import { RpcStxTransferSip9NftContainer } from './rpc-stx-transfer-sip9-nft-container';
+
+export const rpcStxTransferSip9NftRoutes = (
+  <Route
+    path={RouteUrls.RpcStxTransferSip9Nft}
+    element={
+      <AccountGate>
+        <LedgerStacksGate>
+          <CurrentStacksAccountLoader>
+            {account => <RpcStxTransferSip9NftContainer account={account} />}
+          </CurrentStacksAccountLoader>
+        </LedgerStacksGate>
+      </AccountGate>
+    }
+  >
+    <Route index element={<RpcStxTransferSip9Nft />} />
+    <Route path={RouteUrls.FeeEditor} element={<FeeEditor />} />
+    <Route path={RouteUrls.NonceEditor} element={<NonceEditor />} />
+    <Route path={RouteUrls.BroadcastError} element={<BroadcastErrorSheet />} />
+    {ledgerStacksTxSigningRoutes}
+  </Route>
+);
```

Comment:
could this be refactored to share route defs with other similar flows?

Reply:
Yeah, I'm thinking quite a bit might end up being shared b/w these now. It is likely even just one context could be used. I mentioned in the description I'd come back to it, but maybe I'll just do now before merging this one.